DataFormats/Detectors/TRD/src/Tracklet64.cxx

```diff
@@ -43,6 +43,35 @@ GPUd() int Tracklet64::getPadCol() const
   return CAMath::Nint(6.f + mcmCol * ((float)NCOLMCM) + padLocal);
 }
 
+GPUd() float Tracklet64::PadPositionMCM() const
+{
+  return 12.0 - (getPositionBinSigned() * GRANULARITYTRKLPOS);
+}
+
+GPUd() float Tracklet64::PadPosition() const
+{
+  // int padLocalBin = tracklet.getPosition() ^ 0x80;
+  float padMCM = PadPositionMCM();
+  int mcmCol = HelperMethods::getMCMCol(getROB(), getMCM());
+  return float((mcmCol + 1) * NCOLMCM) + 2.0 - padMCM;
+}
+
+GPUd() float Tracklet64::UncalibratedPad() const
```

Comment:
isn't this a duplicate of `getUncalibratedY()`?

Reply:
```UncalibratedPad()``` gives the uncalibrated pad number corresponding to the tracklet position, while ```getUncalibratedY()``` gives the uncalibrated y-coordinate. That's why for ```UncalibratedPad()```, we divide by the pad width, isn't it?

Reply:
but the uncalibrated pad number should really be given by `getPadCol()`. I just want to avoid any confusion and that we use consistently the functions we have in all the places. Otherwise it get's quite messy. So maybe you could use the functions in your PR to the TRD repo which are already available and drop the ones which are added to Tracklet64? The additional function in HelperMethods is fine of course

Reply:
It looks like this essentially does the same thing as PadPosition() discussed above, but in a different way (i.e. calculating the pad from the y coordinate, rather than from MCM and position within the MCM). 
I don't remember exactly, but I might have compared the two to make sure I get it right.